src/router/trie-router/node.ts

```diff
@@ -151,8 +154,10 @@ export class Node<T> {
           // '/hello/*/foo' => match /hello/bar/foo
           if (pattern === '*') {
             const astNode = node.children['*']
-            if (astNode) {
-              handlerSets.push(...this.#gHSets(astNode, method, node.params, Object.create(null)))
+            if (node.children['*']) {
```

Comment:
Is this intended? `node.children['*']` can be `astNode`.

Reply:
Sorry, I made a mistake in changing it.